tests/unit/backend/data/test_DataFactoryService.py

```diff
@@ -88,17 +89,10 @@ def tearDown(self):
         del self.instance
         return super().tearDown()
 
-    def test_fileExists_yes(self):
-        # create a temp file that exists, and verify it exists
-        with tempfile.NamedTemporaryFile(suffix=".biscuit") as existent:
-            assert DataFactoryService().fileExists(existent.name)
-
-    def test_fileExists_no(self):
-        # assert that a file that does not exist, does not exist
-        with tempfile.TemporaryDirectory() as tmpdir:
-            nonexistent = tmpdir + "/0x0f.biscuit"
-            assert not os.path.isfile(nonexistent)
-            assert not DataFactoryService().fileExists(nonexistent)
+    def test_fileExists(self):
```

Comment:
Why did you change these tests?

Reply:
These changes make them consistent with the implementation style of the other tests in `test_DataFactoryService`.  Why I ended up needing to mess with these two tests specifically is a good question, but there was a reason.
I'd go even further and argue that this method shouldn't even exist, and I'm tempted to "nuke" the whole thing: `pathlib.Path.exists()` is sufficient.

Reply:
My understanding is `fileExists()` is meant to be a data service endpoint as part of the layer strategy.  Does this new test check for existing and non-existing files?  I can't tell at a glance.